RIOT-OS/RIOT
# boards/common/stm32: clean up LED definitions

### Contribution description

Let boards only define the port and pin number of each LEDs. The common definitions in `stm32_leds.h` will provide `LED<x>_ON`, `LED<x>_OFF`, `LED<x>_TOGGLE`, `LED<x>_PIN`, `LED<x>_MASK` and `LED<x>_PORT`.

In addition to code de-duplication, this also makes it easier to use LEDs in GPIO LL, which can be beneficial for super low overhead debugging output - e.g. when a bug is timing sensitive and `DEBUG()` would spent to much time for stdio to reproduce a bug.

### Testing procedure

The LED macros should not change to master. Hence, the (except for debug symbols) the binaries generated should not change compared to master.

One could also test if controlling LEDs still works for all the changed boards...

### Issues/PRs references

None
